build-tools/src/main/kotlin/org/jetbrains/kotlin/bitcode/CompileToBitcode.kt

```diff
@@ -68,7 +68,7 @@ open class CompileToBitcode @Inject constructor(
                     // Used flags provided by original build of allocator C code.
                     listOf("-std=gnu11", "-O3", "-Wall", "-Wextra", "-Werror")
                 Language.CPP ->
-                    listOfNotNull("-std=c++14", "-Werror", "-O2",
+                    listOfNotNull("-std=c++17", "-Werror", "-O2",
```

Comment:
Shall we switch to C++17 everywhere for consistency?
```
CAdapterCompile.kt
backend.native/llvm.def
llvmCoverageMappingC/build.gradle
llvmDebugInfoC/build.gradle
llvmDebugInfoC/src/main/cpp/DebugInfoC.cpp
```

Reply:
Right, thanks!